### ISSUE TITLE
Unable to add emulated cards to apple wallet

### ISSUE THREAD

**Issue by tester013** [Aemulo/Release#28]
The reading and emulating function works correctly, but when I tried to add to apple wallet, it always shows "503 Error Downloading Pass", however my network also works fine. At the top of info page there's always a "checking..." hint.
I'm running on iOS 16.5 with Dopamine 2.1.7.11 Roothide. Is this Roothide's problem? Thanks.

**Comment by Arlia-Etoile** [comment]
I dont think it is Roothide's issue, because I couldnt add card either on dopamine2 rootless for the same reason
